third_party/move/move-compiler-v2/tests/reference-safety/v1-tests/mutate_field_invalid.exp

```diff
@@ -1,22 +1,49 @@
 
 Diagnostics:
-error: cannot write to reference in local `s` which is still borrowed
+error: cannot copy mutable reference in local `s` which is still borrowed
```

Comment:
This message is not right.  local s is not being copied, it's being overwritten.  *Please* file an issue to  address this and/or merge it with the following error message.